socorro/app/fetch_transform_save_config.py

```diff
@@ -0,0 +1,18 @@
+"""Default configuration values used by the FetchTransformSaveApp subclasses."""
+from configman.dotdict import DotDict
+
+
+# Ignore non-config module members like DotDict
+always_ignore_mismatches = True
+
+
+# Crash storage source
+source = DotDict({
+    'crashstorage_class': 'socorro.external.fs.crashstorage.FSPermanentStorage',
+})
+
+
+# Crash storage destinations
+destination = DotDict({
+    'crashstorage_class': 'socorro.external.fs.crashstorage.FSPermanentStorage',
+})
```

Comment:
Because the `klass_config` doesn't walk the MRE, this is all moot and won't come into play. We should nix this and make the values the default values for the config options.